src/app/static/src/tests/reportTests.js

```diff
@@ -95,4 +98,459 @@ describe('report page', () => {
         })
     });
 
+    describe("runReport", () => {
```

Comment:
This is a v long file now, can we actually split the publish and run tests into their own files?

Reply:
(nb I think the `mockAxios` instance has to be shared between all tests, so would then also need to be pulled out into its own file and imported)